src/null-safety/index.md

```diff
@@ -135,22 +135,25 @@ listed by the analyzer.
 
 ### Dart 3 backwards compatibility
```

Comment:
This whole section is being removed in https://github.com/dart-lang/site-www/pull/4808/files#diff-74cb202dbd963f8abf2a2a5cee0226b6396b1852141f6fa57422fbd8b89306a0, so to avoid conflicts please leave it as-is.